panel/src/components/Sections/ModelsSection.vue

```diff
@@ -185,11 +240,42 @@ export default {
 			this.reload();
 		}
 	},
+	created() {
+		this.$events.on("model.update", this.reload);
+		this.$events.on("selecting", this.stopSelectingCollision);
```

Comment:
`Suggestion:` This feels like a hack we won't like down the road. What are you trying to achieve here? That I cannot enter select mode in two sections in parallel, select the same pages, then delete them in one and in therefore the other section's state getting out of sync? If so, I would suggest to rather include in the `reload()` method to reset the selecting state (so `this.stopSelecting()`). Since this will be triggered by the `model.update` event, even if I select too sections in parallel, once I delete the models in one section, the other section gets reset and reloaded.

Reply:
Ok, I didn't remember that feedback and feels odd to me (based on behaviour of other system), but ok if that creates problem with a big part of the audience. But then I think this might be rather a global state in our Panel module? Something just feels very duck-tape about this global event listener. Maybe also that it's just called "selecting" which could mean anything in a number of different contexts.

Reply:
How about we at least make it more specific, e.g. calling the global event `section.selecting`?

Reply:
We do something similar for dropdowns, so I don't think it's that much of a hack to be honest. But I agree that a more specific event name is good here. I just changed that.